Update the WebGL2 test expectations

mmatyas opened the pull request:
It seems the WPT test runner/updater has some troubles picking up successful test results, so I've regenerated the `ini` files manually for the WebGL2 tests.

<!-- Please describe your changes on the following line: -->

cc @jdm @zakorgy 

---
<!-- Thank you for contributing to Servo! Please replace each `[ ]` by `[X]` when the step is complete, and replace `___` with appropriate data: -->
- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors
- [x] These changes do not require tests because they are the tests

<!-- Also, please make sure that "Allow edits from maintainers" checkbox is checked, so that we can help you if you get stuck somewhere along the way.-->

<!-- Pull requests that do not address these steps are welcome, but they will require additional verification as part of the review process. -->

highfive commented:
Heads up! This PR modifies the following files:
 * @jgraham: tests/wpt/webgl/meta/conformance2/context/constants-and-properties-2.html.ini, tests/wpt/webgl/meta/conformance2/glsl3/switch-case.html.ini, tests/wpt/webgl/meta/conformance2/buffers/delete-buffer.html.ini, tests/wpt/webgl/meta/conformance2/glsl3/tricky-loop-conditions.html.ini, tests/wpt/webgl/meta/conformance2/attribs/gl-vertex-attrib.html.ini and 24 more

mmatyas committed:
Update the WebGL2 test expectations

It seems the WPT test runner/updater has some troubles picking up
successful test results, so I've regenerated the `ini` files manually
for the WebGL2 tests.

jdm commented:
@bors-servo r+
Yes, it's a limitation of the khronos test harness integration, unfortunately.

bors-servo commented:
:pushpin: Commit 18d40fb has been approved by `jdm`

<!-- @bors-servo r=jdm 18d40fb191ac6778a413d36a7c79d8ec98c1e55b -->

bors-servo commented:
:hourglass: Testing commit 18d40fb191ac6778a413d36a7c79d8ec98c1e55b with merge a73eb8e596d0d92e34a9e0136aabe55823d8f693...

bors-servo commented:
:broken_heart: Test failed - [status-taskcluster](https://community-tc.services.mozilla.com/tasks/groups/fRU3XgSeRS2uFH6N_JEndQ)

jdm commented:
@bors-servo retry
* #24726

bors-servo commented:
:hourglass: Testing commit 18d40fb191ac6778a413d36a7c79d8ec98c1e55b with merge a1490a184eb4e58e940567d8727867df7d11f28b...

bors-servo commented:
:broken_heart: Test failed - [status-taskcluster](https://community-tc.services.mozilla.com/tasks/groups/eqMPsjHATSiCXAywbxnZGQ)

jdm commented:
@bors-servo retry
* #24726

bors-servo commented:
:hourglass: Testing commit 18d40fb191ac6778a413d36a7c79d8ec98c1e55b with merge 66aebfdbee6ae10fe7ea65224bf202f53b2efceb...

bors-servo commented:
:sunny: Test successful - [status-taskcluster](https://community-tc.services.mozilla.com/tasks/groups/OY5LN_ZtSjegyaL7q4yPeA)
Approved by: jdm
Pushing 66aebfdbee6ae10fe7ea65224bf202f53b2efceb to master...
